var/spack/repos/builtin/packages/samtools/package.py

```diff
@@ -59,3 +59,10 @@ def install(self, spec, prefix):
         else:
             make("prefix=%s" % prefix)
             make("prefix=%s" % prefix, "install")
+        # Install dev headers and libs for legacy apps depending on them
+        mkdir = which('mkdir')
```

Comment:
This line isn't needed, Spack already has a `mkdir` function that wraps around `os.mkdir`. This is more platform independent.